charts/argo-rollouts/Chart.yaml

```
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: "0.7"
 description: A Helm chart for Argo Rollouts
 name: argo-rollouts
-version: 0.1.1
+version: 0.1.2
```

```suggestion
version: 0.2.0
```

This is a feature add. How about incrementing the minor version instead of the patch?